.vscode/settings.json

```diff
@@ -1,6 +1,70 @@
 {
    "files.associations": {
       "*.hip": "cpp",
-      "*.sycl": "cpp"
+      "*.sycl": "cpp",
+      "memory_resource": "cpp",
```

Comment:
I know, I know... This is exactly what @paulgessinger warned me about... :frowning:

Still, this should not be added into the repository...

Reply:
Ha! I get to say "I told you so!". :D

Reply:
Oh no! I usually use `git add -i` for everything to avoid this, but the one time I commit the crime of `git commit -a` this happend! :sob: